src/backend/jag-efiling-api/src/main/java/ca/bc/gov/open/jagefilingapi/submission/SubmissionApiImpl.java

```diff
@@ -45,7 +45,7 @@ public DocumentApiImpl(
         logger.info("Generate Url Request Recieved");
 
         logger.debug("Attempting to get fee structure for document");
-        Fee fee = feeService.getFee(new FeeRequest(generateUrlRequest.getDocumentMetadata().getType(), generateUrlRequest.getDocumentMetadata().getSubType()));
+        Fee fee = feeService.getFee(new FeeRequest(generateUrlRequest.getSubmissionMetadata().getType(), generateUrlRequest.getSubmissionMetadata().getSubType()));
```

Comment:
Where do we handle the case where this fails? Or later implementation?

Reply:
I would say later. The current version uses the mock which always returns 7.

Reply:
MAKE IT SO